packages/clay-icon/src/ClayIcon.soy

```diff
@@ -20,6 +21,12 @@
 			{/if}
 		"
 
+		{if $focusable}
```

Comment:
Can this be written `focusable="{$focusable == true}"` ?

Reply:
No, the attributes do not allow **bool** to be used ☹️. To get as close to that `focusable="{$focusable == true ? 'true' : 'false'}"`(is that what you meant?). Sounds good to you?

Reply:
Nah, leave it as it was 😄 Thx!